subprojects/emcee/queue-worker-api/src/main/kotlin/com/avito/emcee/worker/SendBucketResultBody.kt

```diff
@@ -21,13 +21,14 @@ public data class SendBucketResultBody(
         @JsonClass(generateAdapter = true)
         public data class UnfilteredResult(
             val testEntry: TestEntry,
-            val testRunResult: List<TestRunResult>
+            val testRunResults: List<TestRunResult>
         ) {
             @JsonClass(generateAdapter = true)
             public data class TestRunResult(
                 /**
                  * in sec
                  */
+                val udid: String,
```

Comment:
user device identifier? Is this term from iOS/macOs world?

Reply:
`unique device id`. This term is from Emcee: 
https://github.com/avito-tech/Emcee/blob/master/Sources/CommonTestModels/UDID.swift

It is used to distinguish the simulators or real devices each other.